### PR TITLE
Add content length to gcp vertex response to avoid chunked transfer

### PR DESCRIPTION
**Pull request by achraf-mer**

### CODE REVIEW

**Review by this — COMMENTED** [review]
@achraf-mer fix lgtm. However, IMO we shouldn't merge this. Chunking is needed for large response payloads.

File: gcp-vertex-ai-mojo-scorer/src/main/java/ai/h2o/mojos/deploy/gcp/vertex/ai/controller/ModelsApiController.java [hunk 1]
```diff
@@ -56,7 +56,9 @@ public ResponseEntity<ScoreResponse> getScore(
       ScoreRequest request = getRestScoreRequest(gcpRequest);
       ScoreResponse response = getGcpScoreResponse(scorer.score(request));
       // return ResponseEntity.ok(scorer.score(request));
-      return ResponseEntity.ok(response);
+      return ResponseEntity.ok().contentLength(
+              new ObjectMapper().writeValueAsString(response).length()
+      ).body(response);
```
Comment (this) [inline gcp-vertex-ai-mojo-scorer/src/main/java/ai/h2o/mojos/deploy/gcp/vertex/ai/controller/ModelsApiController.java:61]: Avoid serializing twice.
```suggestion
      final String responsePayload = new ObjectMapper().writeValueAsString(response);
      return ResponseEntity.ok()
              .contentLength(responsePayload.length())
              .body(responsePayload);
```

Comment (achraf-mer) [reply gcp-vertex-ai-mojo-scorer/src/main/java/ai/h2o/mojos/deploy/gcp/vertex/ai/controller/ModelsApiController.java:61]: can't be a string, need to be an ScoreResponse

**Comment by jakubhava** [comment]
Closing as stale

**Pull request closed without merging**